mobilecoind-json/src/data_types.rs

```diff
@@ -910,9 +920,17 @@ impl TryFrom<&JsonSignatureRctBulletproofs> for SignatureRctBulletproofs {
         let mut signature = SignatureRctBulletproofs::new();
         signature.set_ring_signatures(RepeatedField::from_vec(ring_sigs));
         signature.set_pseudo_output_commitments(RepeatedField::from_vec(commitments));
-        let proofs_bytes = hex::decode(&src.range_proofs)
+        let range_proof_bytes = hex::decode(&src.range_proof_bytes)
             .map_err(|err| format!("Could not decode from hex: {}", err))?;
```

Comment:
Let's specify which item we couldn't decode.
```suggestion
        let range_proof_bytes = hex::decode(&src.range_proof_bytes)
            .map_err(|err| format!("Could not decode top-level range proof from hex '{}': {}", &src.range_proof_bytes, err))?;
```